x/bep3/client/cli/query.go

```diff
@@ -231,107 +234,108 @@ $ kvcli q bep3 swaps --page=2 --limit=100
 `,
```

Comment:
super nit: I'd move the Example text to the Example field on the command as the other commmands do. It's fine to span several lines.

It messes up autogenerated cli docs otherwise.

Reply:
Do the `status` and `direction` flags need to use the new string names like "SWAP_STATUS_OPEN"? If so we should update the docs here.

Reply:
They use the original values when converting from strings, so it shouldn't require changes to clients

https://github.com/Kava-Labs/kava/blob/91cd58c6ee78214a4eca1866987c071f7b9955f2/x/bep3/types/swap.go#L126-L138